src/osa/configs/sequencer.cfg

```diff
@@ -28,6 +28,7 @@ HIGH_LEVEL_DIR: %(OSA_DIR)s/HighLevel
 LONGTERM_DIR: %(OSA_DIR)s/DL1DataCheck_LongTerm
 MERGED_SUMMARY: %(OSA_DIR)s/Catalog/merged_RunSummary.ecsv
 SEQUENCER_WEB_DIR: %(OSA_DIR)s/SequencerWeb
+GAIN_SELECTION_DIR: %(OSA_DIR)s/GainSel
```

Comment:
With this name, it seems like this is the output directory of the gain selection script instead of the gain selection ending flag